### PR TITLE
[solaredge] Update README.md

### PR DESCRIPTION
**Pull request by PanderMusubi**
Provided essential details for connecting solaredge via token.

### CODE REVIEW

File: addons/binding/org.openhab.binding.solaredge/README.md [hunk 1]
```diff
@@ -16,16 +16,18 @@ Auto-Discovery is not supported, as access requires authentication.
 The following configuration parameters are available for this thing:
 
 - **tokenOrApiKey** (required)
-Either the official API Key for using the public API or when using the inofficial private API: a token which can be retrieved from browser's cookie store when logged into the solaredge website. It is called 'SPRING_SECURITY_REMEMBER_ME_COOKIE'
+Either the [official API Key](https://www.youtube.com/watch?v=iR26nmL5bXg) for using the public API or when using the inofficial private API: a token which can be retrieved from browser's cookie store when logged into the solaredge website. It is called 'SPRING_SECURITY_REMEMBER_ME_COOKIE' When using this token, see also usePrivateApi and meterInstalled.
+
+For Firefox, install an addon such as [Cookie Manager](https://addons.mozilla.org/firefox/addon/a-cookie-manager) to retrieve the token, as this browser no longer offers viewing of cookies with its standard functionality.
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.solaredge/README.md]: Do you really need to install an add-on for this? Can we use the [Storage Inspector](https://developer.mozilla.org/en-US/docs/Tools/Storage_Inspector) of the built-in [Developer Tools](https://developer.mozilla.org/en-US/docs/Tools)?

File: addons/binding/org.openhab.binding.solaredge/README.md [hunk 2]
```diff
@@ -16,16 +16,18 @@ Auto-Discovery is not supported, as access requires authentication.
 The following configuration parameters are available for this thing:
 
 - **tokenOrApiKey** (required)
-Either the official API Key for using the public API or when using the inofficial private API: a token which can be retrieved from browser's cookie store when logged into the solaredge website. It is called 'SPRING_SECURITY_REMEMBER_ME_COOKIE'
+Either the [official API Key](https://www.youtube.com/watch?v=iR26nmL5bXg) for using the public API or when using the inofficial private API: a token which can be retrieved from browser's cookie store when logged into the solaredge website. It is called 'SPRING_SECURITY_REMEMBER_ME_COOKIE' When using this token, see also usePrivateApi and meterInstalled.
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.solaredge/README.md]: You would make me happy if you put each sentence in a new line. I will be much better readable for future maintenance.

**Comment by kaikreuzer** [comment]
@PanderMusubi Did you see @cweitkamp's comments? Would be great if you could address them!

**Comment by PanderMusubi** [comment]
@cweitkamp @kaikreuzer, thanks for your feedback. I'm not familiar with Storage Inspector or the built-in Developer Tools in this context. If there is a way to use them, please add them also as a way to retrieve the data needed. Firefox out of the box cookie management no longer offers viewing contents of cookies, hence the use of that add-on, but probably alternative ways exist.

The text I have added would, in my humble opinion, not be split up into further lines, as the information concerns the same point. It is also a minimum amount of text, which got added. It looks longer here as the URL is seen, which is hidden when this file is rendered. Please add line breaks where you see fit.

**Comment by cweitkamp** [comment]
@PanderMusubi I think there is a misunderstanding. It is not about shortening or splitting your sentences into pieces. They are fine. It is about maintainability. If you add a line break after a fullstop it will be better readable. It does not affect the rendering at all.

**Comment by PanderMusubi** [comment]
Ah, now I get it. Could you do that for me this one time? I no longer have my fork from this repo as I am not a regular contributor to this at all.

**Comment by cweitkamp** [comment]
Superseded by #4727.

**Pull request closed without merging**